packages/cosmic-proto/package.json

```diff
@@ -148,9 +148,9 @@
     "!CHANGELOG.md"
   ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
+    "build": "tsc --project tsconfig.build.json && node scripts/minify.cjs",
     "clean": "rimraf dist",
-    "codegen": "./update-protos.sh && node scripts/codegen.cjs",
+    "codegen": "./update-protos.sh && scripts/codegen.cjs",
```

Comment:
erroneous change ?

```suggestion
    "codegen": "./update-protos.sh && node scripts/codegen.cjs",
```

Reply:
the script has `+x` and a shebang so doesn't need `node `.

but I concur with the above that it should be left out of this Endo sync PR now that it's not a blocker